frames/src/main/java/com/checkout/frames/component/expirydate/usecase/ValidateExpiryDateUseCase.kt

```diff
@@ -0,0 +1,48 @@
+package com.checkout.frames.component.expirydate.usecase
+
+import com.checkout.frames.base.usecase.UseCase
+import com.checkout.frames.base.validation.ValidateResult
+import com.checkout.frames.utils.constants.InputFieldConstants.EXPIRY_DATE_MAXIMUM_LENGTH
+import java.util.Calendar
+
+internal class ValidateExpiryDateUseCase : UseCase<StringBuilder, ValidateResult<String>> {
```

Comment:
Could you please help me to understand how this usecase is used separately from ExpiryDateValidator (headless SDK, part of CardValidator)?
Is there any duplication logic?
Could we combine them?

Reply:
Sure. This use case implements expiry date component logic. For instance, restrict users to add past dates, appending 0 to a month which is greater than 1. There is no duplication logic just restricting users to add past date. We can discuss it on call shall we combine or not.